polkadot-protocol/architecture/system-chains/asset-hub.md

```diff
@@ -1,4 +1,204 @@
+
+- The management team
+- The total supply
+- The number of accounts holding the asset
+- Sufficiency for account existence - whether the asset alone is enough to maintain an account without a native token balance
+- The metadata of the asset, including its name, symbol, and the number of decimals for representation
+
+Some assets can be regarded as sufficient to maintain an account's existence, meaning that users can create accounts on the network without needing a native token balance (i.e., no existential deposit required). Developers can also set minimum balances for their assets. If an account's balance drops below the minimum, the balance is considered dust and may be cleared.
+
+## Assets Pallet
+
+The Polkadot SDK's Assets pallet is a powerful tool for creating and managing fungible asset classes with a fixed supply. It offers a secure and flexible way to issue, transfer, freeze, and destroy assets. The pallet supports various operations and includes permissioned and non-permissioned functions to cater to simple and advanced use cases.
+
+For full details, see the [assets pallet repository](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/assets){target=\_blank}, which contains reference documentation and examples of how to interact with the pallet.
+
+### Key Features
+
+Key features of the Assets pallet include:
+
+- **Asset issuance** - allows the creation of a new asset, where the total supply is assigned to the creator's account
+- **Asset transfer** - enables transferring assets between accounts while maintaining a balance in both accounts
+- **Asset freezing** - prevents transfers of a specific asset from one account, locking it from further transactions
+- **Asset destruction** - allows accounts to burn or destroy their holdings, removing those assets from circulation
+- **Approval-based transfers** - a non-custodial mechanism to enable one account to approve a transfer of assets on behalf of another
+
+### Main Functions
+
+The Assets pallet provides a broad interface for managing fungible assets. Some of the main dispatchable functions include:
+
+- **`create()`** - create a new asset class by placing a deposit, applicable when asset creation is permissionless
+- **`issue()`** - mint a fixed supply of a new asset and assign it to the creator's account
+- **`transfer()`** - transfer a specified amount of an asset between two accounts
+- **`approve_transfer()`** - approve a non-custodial transfer, allowing a third party to move assets between accounts
+- **`destroy()`** - destroy an entire asset class, removing it permanently from the chain
+- **`freeze()`** and **`thaw()`** - administrators or privileged users can lock or unlock assets from being transferred
+
+For a full list of dispatchable and privileged functions, see the [Call Enum Rust docs](https://docs.rs/pallet-assets/latest/pallet_assets/pallet/enum.Call.html){target=\_blank}.
+
+### Public Functions
```

Comment:
I'm not sure if we should use the term "public functions" - maybe "queryable", or something along those lines? They're functions that query chain state / storage mappings